vars/sectionDeployToAKS.groovy

```diff
@@ -154,9 +187,19 @@ def call(params) {
               }
             }
           }
+          if (testLabels.contains('enable_full_functional_tests')) {
```

Comment:
if this label is enabled then "Functional Test" should be disabled

I think it also needs an `onFunctionalTestEnvironment` block added around it here